field_manager/fm_yaml.F90

```diff
@@ -31,493 +31,320 @@
 !> @{
 module fm_yaml_mod
 #ifdef use_yaml
+
 use yaml_parser_mod
+use mpp_mod, only: mpp_error, fatal
 implicit none
 private
 
-integer :: i, table_i, type_i, model_i, var_i, var_j, attr_j !< counters
-
-!> @}
-! close documentation grouping
+public :: build_fmTable
 
-!> @brief This type represents the subparameters for a given variable parameter.
-!> This type contains the name of the associated parameter, the key / value pairs for this subparameter,
-!! and the following methods: getting names and properties, and self destruction.
-!> @ingroup fm_yaml_mod
```

Comment:
Any types or interfaces should have the `@ingroup fm_yaml_mod` comment to add it to the module page.

Reply:
Added before all type definitions in 02085437c8de6c6d895c878158f2186edf07eacd.